src/main/java/bio/terra/Main.java

```diff
@@ -32,4 +45,40 @@ public int getExitCode() {
       return 10;
     }
   }
+
+  // Mitigation for Spring Core Bug:
+  // https://spring.io/blog/2022/03/31/spring-framework-rce-early-announcement
+  @Bean
+  public WebMvcRegistrations mvcRegistrations() {
+    return new WebMvcRegistrations() {
+      @Override
+      public RequestMappingHandlerAdapter getRequestMappingHandlerAdapter() {
+        return new ExtendedRequestMappingHandlerAdapter();
+      }
+    };
+  }
+
+  private static class ExtendedRequestMappingHandlerAdapter extends RequestMappingHandlerAdapter {
+
+    @Override
+    protected InitBinderDataBinderFactory createDataBinderFactory(
+        List<InvocableHandlerMethod> methods) {
+
+      return new ServletRequestDataBinderFactory(methods, getWebBindingInitializer()) {
+
+        @Override
+        protected ServletRequestDataBinder createBinderInstance(
+            @Nullable Object target, String name, NativeWebRequest request) throws Exception {
+
+          ServletRequestDataBinder binder = super.createBinderInstance(target, name, request);
+          String[] fields = binder.getDisallowedFields();
+          List<String> fieldList =
+              new ArrayList<>(fields != null ? Arrays.asList(fields) : Collections.emptyList());
+          fieldList.addAll(Arrays.asList("class.*", "Class.*", "*.class.*", "*.Class.*"));
+          binder.setDisallowedFields(fieldList.toArray(new String[] {}));
```

Comment:
May as well use current JDK APIs here. Also the use of the ternary op here doesn't seem necessary.
```suggestion          
          List<String> fieldList = new ArrayList<>(List.of("class.*", "Class.*", "*.class.*", "*.Class.*"));
          String[] fields = binder.getDisallowedFields();
          if (fields != null) {
            fieldList.addAll(List.of(fields);
          }
          binder.setDisallowedFields(fieldList.toArray());
```